mesheryctl/internal/cli/root/system/config.go

```diff
@@ -61,24 +61,37 @@ func getContexts(configFile string) ([]string, error) {
 	}
 
 	log.Debugf("Get context API response: %s", string(body))
-	var results []map[string]interface{}
+	var results map[string]interface{}
 	err = json.Unmarshal(body, &results)
 	if err != nil {
 		return nil, err
 	}
 
-	if results == nil {
+	if results == nil || results["contexts"] == nil {
 		errstr := "Error unmarshalling the context info, check " + configFile + " file"
 		return nil, errors.New(errstr)
 	}
-
-	log.Debugf("Unmarshalled get context API response: %s", results)
-	var contexts []string
-	for _, item := range results {
-		contexts = append(contexts, item["name"].(string))
+	contexts, ok := results["contexts"].([]interface{})
```

Comment:
Assuming that the API returns an array even if there is a single context.

Reply:
If there is a single context then the array will have one item